_docs-sources/iac/whats-this/index.md

```diff
@@ -1,5 +1,25 @@
 # What is the Infrastructure as Code Library?
 
-Qu'est-ce que c'est?
+The Gruntwork Infrastructure as Code Library (IaC Library) is a collection of reusable code that enables you to deploy and manage infrastructure quickly and reliably. We've taken the thousands of hours we spent building infrastructure on AWS and condensed all that experience and code into pre-built packages or modules.  
```

Comment:
:thinking: one phrase that gave me pause was "reusable code" vs "reusable Terraform modules" - not sure what is more clear or helpful. Technically we have all kinds of code blended together, so your current phrasing is probably more accurate.